org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/EclipseSbtBuildManager.scala

```diff
@@ -179,10 +184,11 @@ class EclipseSbtBuildManager(val project: IScalaProject, settings0: Settings) ex
   }
 
   // take by-name argument because we need incOptions only when we have a cache miss
-  private[zinc] def latestAnalysis(incOptions: => IncOptions): Analysis =
+  def getLatestAnalysis(incOptions: => IncOptions): Analysis =
```

Comment:
In scala we don't usually use the Java `get` prefix. In this case, I think `latestAnalysis` was good enough.